v19.2/alter-type.md

```diff
@@ -31,7 +35,7 @@ The user must have the `CREATE` [privilege](authorization.html#assign-privileges
 |-----------|-------------
 | `table_name` | The name of the table with the column whose data type you want to change.
 | `column_name` | The name of the column whose data type you want to change.
-| `typename` | The new [data type](data-types.html) you want to use.
+| `typename` | The new data type you want to use.<br>`ALTER TABLE ... ALTER TYPE` and [`ALTER TABLE ... ALTER COLUMN ... SET DATA TYPE`](alter-column.html) are aliases.
```

Comment:
I think `ALTER TABLE ... ALTER TYPE` should be replaced with `ALTER TABLE ... ALTER ... TYPE` since a col name should go between.
Applies to the other pages as well

Also [SET DATA] is optional to provide.